internal/pkg/dashboard/components/footer.go

```diff
@@ -19,13 +18,12 @@ import (
 type Footer struct {
 	tview.TextView
 
-	nodes             []string
-	selectedNodeIndex int
+	selectedNode string
```

Comment:
Moved the node selection logic from the footer to the dashboard.go itself & made it simpler.